src/java.desktop/share/classes/javax/swing/ImageIcon.java

```diff
@@ -61,6 +61,11 @@
  * from Images. Images that are created from a URL, filename or byte array
  * are preloaded using MediaTracker to monitor the loaded state
  * of the image.
+ * If the image source parameter to a constructor or method is non-null,
```

Comment:
FWIW when I first suggested this statement I said it needed to cover the setter too
'We probably should add the similar text on setImage().'
I guess we can keep it here now it's here.